nemo_aligner/data/nlp/datasets.py

```diff
@@ -331,12 +331,13 @@ def __getitem__(self, idx):
 
         # ignore the example whose tokenized text exceeds max seq length.
         if max_curr_seq_len > self.seq_length:
-            chosen_tokens = chosen_tokens[: self.seq_length]
-            rejected_tokens = rejected_tokens[: self.seq_length]
+            nograd_length = 32
```

Comment:
Can you please move this to the constructor and call it self.nograd_length?

Reply:
Done in the latest commit.